evalml/model_understanding/graphs.py

```diff
@@ -449,7 +449,9 @@ def partial_dependence(pipeline, X, features, grid_resolution=100):
             If features is an int, it must be the index of the feature to use.
             If features is a string, it must be a valid column name in X.
             If features is a tuple of int/strings, it must contain valid column integers/names in X.
-        grid_resolution (int): Number of samples of feature(s) for partial dependence plot
+        grid_resolution (int): Number of samples of feature(s) for partial dependence plot.  If this value
+            does not exceed the maximum number of categories present in categorical data within X, it will be
```

Comment:
If this number does not exceed or if this number does exceed?

Reply:
I think the wording could be clarified a bit. If grid_resolution is less than the maximum number of..etc`